### PR TITLE
CNV-11201: RN Roll out of node network configuration policy

### PR DESCRIPTION
**Pull request by sjhala-ccs**
[CNV-11201](https://issues.redhat.com/browse/CNV-11201)

Preview: https://deploy-preview-42355--osdocs.netlify.app/openshift-enterprise/latest/virt/virt-4-10-release-notes.html#virt-4-10-networking-new

### CODE REVIEW

**Comment by sjhala-ccs** [comment]
@rhrazdil Can you please review? Thank you.

**Review by rhrazdil — COMMENTED** [review]
@sjhala-ccs The release note looks good.

But I also read the section that the release note refers to and one thing that's odd there is the Example Output at the end of this section: https://deploy-preview-42355--osdocs.netlify.app/openshift-enterprise/latest/virt/node_network/virt-updating-node-network-config.html#virt-creating-interface-on-nodes_virt-updating-node-network-config

There is:
2. Create the node network policy:
```
$ oc apply -f br1-eth1_policy.yaml
```

Example output
```
NAME                     STATUS
node01.br1-eth1-policy   Pending
node02.br1-eth1-policy   Progressing
node03.br1-eth1-policy   Available
node04.br1-eth1-policy   Progressing
node05.br1-eth1-policy   Progressing
node06.br1-eth1-policy   Pending
```

I'n not sure the user know at this moment, how to get to the example output? From the article, is appears as if the output would be produced by the `oc apply` command, but it is actually obtained by running `oc get nnce`, but IIUC, that is explained only in the following section.

**Comment by sjhala-ccs** [comment]
@rhrazdil I would have to create a separate PR against the `main` branch to update the section that the release note refers to. For tracking purposes, can you please open a documentation BZ for this? Do you want me to move the `Example output` code block to the `Confirming node network policy updates on nodes` section or remove it completely from the documentation?

File: virt/virt-4-10-release-notes.adoc [hunk 1]
```diff
@@ -68,6 +68,7 @@ The SVVP Certification applies to:
 === Networking
 
 //CNV-11201 Rollout of NodeNetworkConfigurationPolicy
+* You can now xref:../virt/node_network/virt-updating-node-network-config.adoc#virt-creating-interface-on-nodes_virt-updating-node-network-config[configure multiple nmstate-enabled nodes concurrently] by using a single `NodeNetworkConfigurationPolicy` manifest.
```
Comment (yossisegev) [inline virt/virt-4-10-release-notes.adoc:72]: Why does the xref take to the `Creating an interface on nodes` section?
Based on the sentence given here, seems to me it should lead to the `Updating node network configuration
`, which is the actually the header of the entire page.
WDYT?

**Review by yossisegev — CHANGES_REQUESTED** [review]

File: virt/virt-4-10-release-notes.adoc (hunk 1 above)
Comment (yossisegev) [reply virt/virt-4-10-release-notes.adoc:72]: Sorry, I didn't read the entire paragraph.
LGTM

**Comment by ousleyp** [comment]
LGTM!

**Comment by bobfuru** [comment]
In preparation for OCP 4.10 GA, I'm moving the Milestone to "Next Release" as this PR did not make the merge by the 4.10 GA date. Post-GA, any open 4.10 PRs fall under the "Next Release" bucket in the same way that 4.6-4.9 PRs are already there. This change does not have any impact on the work in this PR; it's a housekeeping task to keep account of all PRs that had already merged by the 4.10 GA date. 😁

**Pull request merged**